server/README.md

```diff
@@ -0,0 +1,37 @@
+# GPay Group Buy Server
+
+This is the Group Buy server which serves the web client through REST API.
+
+## Prerequisites
+Install [Node 10](nodejs.org) and [Yarn](classic.yarnpkg.com/en/docs/install/).
+
+## Development
+
+In the project directory, you can run:
+
+### `yarn`
+This will install all of the dependencies you need.
+
+### `yarn start`
+
+Runs the app in the development mode.<br />
+Open [http://localhost:5000](http://localhost:5000) to view it in the browser.
+
+The page will reload if you make edits.<br />
+You will also see any lint errors in the console.
+
+### `yarn lint`
```

Comment:
will add that this is about gts as commented by yifu in https://github.com/googleinterns/gpay-group-buy/pull/21#discussion_r439259564